packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -141,63 +136,137 @@ export class FacetSearch {
         class={
           'search-input placeholder-on-background-variant flex-grow outline-none focus:outline-none mx-2'
         }
-        placeholder={this.props.controller.strings.placeholder()}
-        value={this.props.controller.state.facetSearchQuery}
+        placeholder={this.strings.placeholder()}
+        value={this.component.facetSearchQuery}
       />
     );
   }
 
   private highlightSuggestion(suggestion: string) {
-    const search = regexEncode(this.props.controller.state.facetSearchQuery);
+    const sanitizedSuggestion = sanitize(suggestion);
+    if (this.component.facetSearchQuery.trim() === '') {
+      return sanitizedSuggestion;
+    }
+
+    const search = regexEncode(this.component.facetSearchQuery);
     const regex = new RegExp(`(${search})`, 'ig');
     return sanitize(suggestion).replace(regex, '<b>$1</b>');
   }
 
+  private get suggestionClasses() {
+    return 'suggestion cursor-pointer px-2 py-1 text-sm';
+  }
+
   private get suggestions() {
-    return (this.props.controller.facetSearchState
-      .values as BaseFacetSearchResult[]).map((suggestion, index) => {
-      return (
-        <li
-          onClick={() => this.onSelectValue(suggestion)}
-          onMouseDown={(e) => e.preventDefault()}
-          part="suggestion"
-          class="suggestion cursor-pointer flex flex-row items-center px-2 text-sm"
-          value={index}
-        >
+    return this.values.map((suggestion, index) => (
+      <li
+        onClick={() => this.onSelectValue(suggestion)}
+        onMouseDown={(e) => e.preventDefault()}
```

Comment:
I think it's in order not to trigger the blur

Reply:
@kaitlynkatz, perhaps this pattern could be helpful to you?

When a search box suggestion is clicked, use the `mousedown` event to prevent the input blur, while still using a click event for selecting a suggestion.